app/_src/gateway/upgrade-v2.md

```diff
@@ -364,3 +364,72 @@ Perform a rolling upgrade of your cluster:
    is going smoothly.
 8. When your traffic is fully migrated to the new cluster,
    decommission your old data planes.
+
+
+## Upgrade to {{site.base_gateway}} 3.x.x and retain 2.x.x alerts for Prometheus
```

Comment:
@mheap I added this section based on the conversation we had in Slack the other day. Is this accurate and kind of what you were thinking?